docs/openstack/advanced-openstack-topics/data-transfer/data-transfer-from-to-vm.md

```diff
@@ -0,0 +1,568 @@
+        - **all Ubuntu images**: ubuntu
+        - **all CentOS images**: centos
+        - **all Rocky Linux images**: centos
+        - **all Fedora images**: fedora
+        - **all Debian images**: debian
+        - **all RHEL images**: cloud-user
+
+    **"Password"**: "`<Leave blank as you using SSH key>`"
+
+- Change Authentication Options
+
+Before saving, click the "Advanced" button.
+In the Advanced Site Settings, under SSH >> Authentication settings check
+“Allow agent forwarding” and select Private key file with `.ppk` extension from
+the file picker.
+
+![Advanced Site Settings for SSH Authentication](images/winscp-ssh-auth.png)
+
+!!! tip "Helpful Tip"
+    You can save your above configured site with some preferred name by
+    clicking the "Save" button and then giving a proper name to your site.
+    So that next time you don't need to again manually enter all your configuration.
+    ![Save Site WinSCP](images/winscp-save-site.png)
+
+#### Using WinSCP
+
+You can follow above step to manually add a new site next time you open WinSCP
+or, you can connect to your previously saved site (as listed on popup dialog
+will show your all saved site name list) that will show up by just clicking on
+the site name.
+
+Then click "Login" button to connect to your NERC project's VM as shown below:
+
+![Login](images/winscp-site-login.png)
+
+![Successful connection](images/winscp-site-successfully-connected.png)
+
+You should now be connected to the VM's remote directories/files. You can drag
+and drop your files to/from file windows to begin transfer. When done, click the
+X icon up top right to disconnect.
+
+### ii. Cyberduck
+
+[Cyberduck](https://docs.cyberduck.io/cyberduck/) is a libre server and cloud
+storage browser for Mac and Windows. With an easy-to-use interface, connect to
+servers, enterprise file sharing, and cloud storage.
+
+- **Prerequisites**
+
+- Cyberduck installed, see [Download and Install the latest version of the Cyberduck](https://cyberduck.io/download/)
+for more information.
+
+#### Configuring Cyberduck
+
+- Click on "Open Connection" tab button as shown below:
+
+![Open Connection](images/cyberduck-open-connection-new.png)
+
+- Select either **"SFTP"** or **"FTP"** from the dropdown options as shown below:
+
+![Choose Amazon S3](images/cyberduck-select-sftp-or-ftp.png)
+
+- Provide the following required information:
+
+    **"Server"**: "`<Your Floating IP of VM>`"
+
+    **"Port"**: "22"
+
+    **"User name"**: "`<Default User name based on OS>`"
+
+    !!! info "Default User name based on OS"
+        - **all Ubuntu images**: ubuntu
+        - **all CentOS images**: centos
+        - **all Rocky Linux images**: centos
+        - **all Fedora images**: fedora
+        - **all Debian images**: debian
+        - **all RHEL images**: cloud-user
+
+    **"Password"**: "`<Leave blank as you using SSH key>`"
+
+    **"SSH Private Key"**: "Choose the appropriate SSH Private Key from you local
```

Comment:
from your local machine that has the corresponding public key attached to your VM